apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx

```diff
@@ -14,19 +14,27 @@ interface IFluentDocsContainerProps {
 export const FluentDocsContainer: React.FC<IFluentDocsContainerProps> = ({ children, context }) => {
   // eslint-disable-next-line deprecation/deprecation
   const selectedTheme = themes.find(theme => theme.id === context.globals[THEME_ID]);
+
+  const DefaultDocs = <DocsContainer context={context}>{children}</DocsContainer>;
+
   return (
-    <>
-      <FluentProvider style={{ backgroundColor: 'transparent' }} theme={selectedTheme?.theme ?? webLightTheme}>
-        <FluentDocsHeader
-          // eslint-disable-next-line deprecation/deprecation
-          storybookGlobals={context.globals}
-        />
-      </FluentProvider>
+    <div className={context.parameters.fullScreen && 'full-screen'}>
```

Comment:
this component has lot of unrelated logic. we should extract that to single source of truth. something like following:

```tsx
export const FluentDocsContainer: React.FC<IFluentDocsContainerProps> = ({ children, context }) => {
  const content =
    context.parameters.hideThemeSelector === true ? (
      <DocsContainer context={context}>{children}</DocsContainer>
    ) : (
      <EnhancedDocsContainer context={context}>{children}</EnhancedDocsContainer>
    );

  return <div className={context.parameters.fullScreen && 'full-screen'}>{content}</div>;
};

const EnhancedDocsContainer: React.FC<IFluentDocsContainerProps> = props => {
  const { context, children } = props;
  const selectedTheme = themes.find(theme => theme.id === context.globals[THEME_ID]);
  return (
    <>
      <FluentProvider style={{ backgroundColor: 'transparent' }} theme={selectedTheme?.theme ?? webLightTheme}>
        <FluentDocsHeader storybookGlobals={context.globals} />
      </FluentProvider>

      <FluentProvider style={{ backgroundColor: 'transparent' }} theme={webLightTheme}>
        <DocsContainer context={context}>{children}</DocsContainer>
      </FluentProvider>
    </>
  );
};

```